python/cm/migrations/0081_ansible_venv.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.7 on 2021-12-19 17:23
```

Comment:
I think this migration file is not needed

Reply:
Migrations has been merged.

Reply:
Yes, I will fix it